docs/DeveloperGuide.md

```diff
@@ -323,14 +323,15 @@ Deletion of `Person` from NetConnect is facilitated by `Model#getPersonById(Id)`
 * **Alternative 1 (current choice):** A single `Model#deletePerson(Person)` method.
   * Pros:
     1. Implementation of `Model#getPersonById(Id)` and `Model#getPersonByName(Name)` can be reused for other purposes.
-    1. Simple implementation of `Model#deletePerson(Person)`.
+    2. Simple implementation of `Model#deletePerson(Person)`.
   * Cons: Need to ensure `Person` with matching `Id` or `Name` exists in the list before `Model#getPersonById(Id)`, `Model#getPersonByName(Name)` and `Model#deletePerson(Person)` are called.
 
 * **Alternative 2:** Separate methods of `Model#deletePersonById(Id)` and `Model#deletePersonByName(Name)`.
   * Pros: Simple implementation of `DeleteCommand#execute(...)`.
   * Cons:
     1. Presence checks required in `Model#deletePersonById(Id)` and `Model#deletePersonByName(Name)`.
-    1. Much boilerplate code since `Model#deletePersonById(Id)` and `Model#deletePersonByName(Name)` are very similar.
+    2. More boilerplate code since `Model#deletePersonById(Id)` and `Model#deletePersonByName(Name)` are very similar.
```

Comment:
Numbering is intentional as per [Markdown Coding Standard](https://se-education.org/guides/conventions/markdown.html)!